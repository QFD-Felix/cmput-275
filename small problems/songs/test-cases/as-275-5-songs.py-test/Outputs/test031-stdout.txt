23 8 29 23 63 8 38 9 4 19 67 3 15 8 20 5 2 0 34 6 43 9 13 18 8 15 13 22 17 24 53 6 6 3 9 39 11 5 12 15 36 7 49 12 2 13 15 4 92 8 35 13 10 13 56 14 0 16 2 13 28 18 13 2 128 49 2 19 57 2 0 38 25 9 4 6 4 35 12 14 19 67 50 4 0 10 3 13 6 16 17 5 77 62 4 26 5 6 6 64 11 30 0 23 23 74 0 45 5 27 13 22 10 2 3 1 2 2 2 11 33 8 16 5 0 73 8 14 16 10 15 57 3 10 3 7 11 101 16 10 21 17 24 24 6 59 0 13 3 94 6 38 20 67 3 9 79 8 1 3 11 15 75 23 105 8 11 1 6 1 13 11 6 36 16 0 51 40 17 2 0 6 3 1 7 15 13 41 13 5 6 6 1 37 14 77 10 6 20 3 11 6 3 10 10 14 4 5 46 6 22 12 14 12 20 52 0 25 6 12 1 12 0 48 20 4 22 18 13 39 43 5 34 22 43 11 29 16 1 1 44 10 17 6 13 11 6 0 66 1 24 10 0 5 55 5 0 6 42 11 4 1 61 17 48 16 10 11 21 19 13 4 15 6 11 15 39 9 4 17 23 17 19 8 27 0 6 12 0 13 3 3 21 15 1 2 0 0 0 7 5 18 0 4 0 5 10 13 9 13 0 17 0 13 10 10 13 0 0 2 0 0 0 0 3 0 0 0 0 1 1 0 0
