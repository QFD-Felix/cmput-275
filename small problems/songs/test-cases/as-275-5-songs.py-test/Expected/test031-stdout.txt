23 8 29 23 63 8 38 9 4 19 67 3 15 8 20 5 2 0 34 6 43 9 13 18 8 15 13 22 17 24 53 6 6 3 9 39 11 5 12 15 36 7 49 12 2 13 15 4 92 8 35 13 10 13 56 14 0 16 2 13 28 18 13 2 129 49 2 19 57 2 0 38 25 9 4 6 4 35 12 14 19 67 50 4 0 10 3 13 6 16 17 5 78 62 4 26 5 6 6 64 11 30 0 23 23 75 0 45 5 27 13 22 10 2 3 1 2 2 2 11 33 8 16 5 0 76 8 14 16 10 15 58 3 10 3 7 11 104 16 10 21 17 24 25 6 62 0 13 3 97 6 39 21 70 3 9 11 8 1 3 12 16 78 24 108 9 12 2 70 6 1 13 11 6 38 16 0 23 42 17 2 0 6 3 1 7 17 11 43 15 5 6 7 1 39 16 79 12 8 3 21 29 3 11 6 3 10 10 14 4 5 46 6 22 12 14 12 20 52 0 25 6 12 1 12 0 48 20 4 22 18 13 39 43 5 34 22 45 11 29 16 1 1 46 10 17 6 13 11 6 0 69 1 24 10 24 5 57 5 0 6 44 11 4 1 67 19 11 18 12 13 23 21 15 6 17 8 13 17 39 0 40 9 4 17 23 17 19 8 28 0 6 12 0 13 3 3 22 15 1 2 0 21 0 7 5 22 20 4 0 6 11 17 11 17 0 0 9 0 16 13 13 18 0 0 0 4 9 0 0 0 0 0 5 0 0 0 0 1 0 1 1 0 0
